hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -140,11 +146,16 @@ public MapContainer(final String name, final Config config, final MapServiceCont
         } else {
             this.globalIndexes = null;
         }
+        this.addEventPublishingEnabled = nodeEngine.getProperties().getBoolean(LOAD_ALL_PUBLISHES_ADD_EVENT);
```

Comment:
Can we switch to `nodeEngine.getProperties().getBoolean(LOAD_ALL_PUBLISHES_ADD_EVENT)` and have it in group properties?